src/components/WebexActivity/WebexActivity.jsx

```diff
@@ -22,13 +23,18 @@ export default function WebexActivity({activityID, className, style}) {
     displayHeader,
     personID,
     text,
+    attachments,
   } = useActivity(activityID);
   const [cssClasses, sc] = webexComponentClasses('activity', className);
+  const isCard = attachments && attachments[0] && attachments[0].contentType === 'application/vnd.microsoft.card.adaptive';
```

Comment:
this logic should be hidden in the adapter, maybe add a 'hasCard' prop on the activity, or even `card` itself